examples/advanced/sklearn-svm/job_configs/sklearn_svm_base/app/custom/svm_assembler.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We will move to this app_common when it gets matured
+from typing import Dict
+
+import numpy as np
+from sklearn.svm import SVC
+
+from nvflare.apis.dxo import DataKind
+from nvflare.app_common.aggregators.assembler import Assembler
+
+
+class SVMAssembler(Assembler):
+    def __init__(self, kernel):
+        super().__init__(data_kind=DataKind.WEIGHTS)
+        # Record the global support vectors
+        # so that only 1 round of training is performed
+        self.support_x = None
+        self.support_y = None
+        self.kernel = kernel
+
+    def get_model_params(self, data: dict):
+        return {"support_x": data["support_x"], "support_y": data["support_y"]}
+
+    def assemble(self, current_round: int, data: Dict[str, dict]) -> dict:
+        if current_round == 0:
```

Comment:
we could possibly get rid of this as well, but we can do that in a follow up enhancement PR.

Reply:
sounds good